changelog/wave/v1.10.2.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Wave v1.10.2
+date: 2024-08-03
+tags: [wave]
+---
+
+## Feature updates and improvements
+
+- Add second level cache in RegistryAuthService by @munishchouhan in https://github.com/seqeralabs/wave/pull/588
```

Comment:
@christopher-hakkaart, should we use code font for `RegistryAuthService`?

Reply:
Good question. I haven't been using code font in the changelog as these are mostly commit or PR messages which haven't been formatted. I think as a rule we can start to code font these to make it clearer to the reader.

Reply:
Fixed in revision